CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 #### New Features
 
+#### Improvements
+- Disables sql simplification when sort is performed after limit. 
+  - Improve performance of generated query since limit stops table scanning as soon as number of record is satisfied.
+  - Note before sort after limit and limit after sort will always return top k result, but now sort after limit will no longer return deterministic result.
```

Comment:
This sentence seems not easy to read, maybe use an example?

Reply:
added an example with the snowpark code